docs/configuration/overview.mdx

```diff
@@ -21,7 +21,7 @@ Payload is a _config-based_, code-first CMS and application framework. The Paylo
 | `serverURL`            | A string used to define the absolute URL of your app including the protocol, for example `https://example.com`. No paths allowed, only protocol, domain and (optionally) port |
 | `collections`          | An array of all Collections that Payload will manage. To read more about how to define your collection configs, [click here](/docs/configuration/collections). |
 | `globals`              | An array of all Globals that Payload will manage. For more on Globals and their configs, [click here](/docs/configuration/globals). |
-| `cors`                 | Either a whitelist array of URLS to allow CORS requests from, or a wildcard string (`'*'`) to accept incoming requests from any domain. |
+| `cors`                 | Configure CORS settings here. [More](#cors) |
```

Comment:
> Configure CORS settings here. 

This is not sufficient. I prefer the old description, but we could make it more generic since you did expand the options of what is possible in the PR. Regardless, what is CORS and why would somebody need to configure it, this question should be answered inline as concise as possible.